buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -488,18 +488,18 @@ class BeamModulePlugin implements Plugin<Project> {
     def jsr305_version = "3.0.2"
     def kafka_version = "2.4.1"
     def nemo_version = "0.1"
-    def netty_version = "4.1.52.Final"
+    def netty_version = "4.1.72.Final"
```

Comment:
As Google Cloud libraries' netty dependencies come from grpc-netty-shaded, this change shouldn't be needed (but I guess this should be no harm)